packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -757,24 +723,29 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
                             notExists: true
                         }];
                     }
-                    const response = [];
-                    const baseName = name + ((!name.length || name.endsWith('/')) ? '' : '/');
-                    res.forEach(arr => {
-                        let entryId = id;
-                        if (arr.isDir) {
-                            if (entryId === '' || entryId === '*') {
-                                entryId = arr.file;
-                                arr.file = '_data.json'; // We return a "virtual file" to mark the directory as existing
-                            } else {
-                                arr.file += '/_data.json'; // We return a "virtual file" to mark the directory as existing
-                            }
+                } catch (err) {
+                    if (!err.toString().endsWith(utils.ERRORS.ERROR_NOT_FOUND)) {
```

Comment:
Can `readDir` throw a string? Or an error?

I would check like this `if (!err.message.contains(...))` that is more resilient if the error is not formatted exactly like you expect.

Reply:
kk